concourse/build-and-deploy.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Which one is meant to be the deploy script? Not sure I understand why there is both `deploy` and `build-and-deploy`, when would each one be called?

Reply:
@nazrhom the publish job has a `deploy-concourse` task which calls `build-and-deploy`. `build-and-deploy` calls `buid.sh` and then `deploy.sh`. I just split `build-and-deploy` in two separate scripts, It could be that's there are only two scripts, `build` and `build-and-deploy`. I don't mind, do you think that would be better better?